build/status/MV9nZXRfY29uZG9yX2RhdGEvb3V0L21vZGVsZWQucmRzLmluZA.yml

```diff
@@ -2,12 +2,12 @@ version: 0.3.0
 name: 1_get_condor_data/out/modeled.rds.ind
 type: file
 hash: bd0fe3d2bdfc77d44e93966f0cf6c811
-time: 2017-12-12 18:12:43 UTC
+time: 2017-12-12 20:55:14 UTC
 depends:
   master_list_orig: 825ed3e7f1684d93571ca64ceda1933b
-  1_get_condor_data/out/master_list.rds.ind: 4ae075cf9f033b14cc57e31d92ef3434
+  1_get_condor_data/out/master_list.rds.ind: 5c16d9edbd3c75725641f4cf80d6185d
 fixed: 39357c2205fe9c1478bf6a122e7ff80d
 code:
   functions:
-    get_data: 424ee55ba94b3ee8c8a1ef89626251ce
+    get_data: 1576ae913a9d39b30e396d6a32c38744
```

Comment:
here's an example of where it's sorta nice to be committing these build/status files. you can see that 1_get_condor_data/out/modeled.rds.ind already existed at 6pm UTC, but 2 hours later you edited `get_data` and updated `1_get_condor_data/out/master_list.rds.ind`, which led to the update of this file even though the arguments in the `command` didn't change (that's the `fixed` field here) and neither did `master_list_orig`. And the result was that this file was rebuilt but didn't actually change contents (that's the `hash` field above), which is also nice to know.